MAUI/NavigationDrawer/Main-Content.md

```diff
@@ -0,0 +1,242 @@
+---
+layout: post
+title: Setting Main Content in .NET MAUI Navigation Drawer control | Syncfusion
+description: Learn here all about Setting Main Content support in Syncfusion .NET MAUI Navigation Drawer (SfNavigationDrawer) control and more.
+platform: maui
+control: NavigationDrawer
+documentation: ug
+---
+
+
+# Setting Main Content in Xamarin Navigation Drawer (SfNavigationDrawer)
+
+Main content of NavigationDrawer is always visible and it can be set using [`ContentView`]() property. In the following code example, [`ContentView`]() is switched when selection changes in ListView.
+
```

Comment:
The main content of the NavigationDrawer is always visible and can be set using the 'ContentView' property. In the code example below, the 'ContentView' is switched when the selection changes in the ListView. In short, this allows for easy navigation and switching of content.

Reply:
Changed